cts-java/src/test/java/org/ga4gh/cts/api/g2p/GenotypePhenotypeSearchIT.java

```diff
@@ -0,0 +1,264 @@
+package org.ga4gh.cts.api.g2p;
+
+import org.apache.avro.AvroRemoteException;
+import org.ga4gh.ctk.transport.GAWrapperException;
+import org.ga4gh.ctk.transport.URLMAPPING;
+import org.ga4gh.ctk.transport.protocols.Client;
+import org.ga4gh.cts.api.TestData;
+import org.ga4gh.cts.api.Utils;
+import org.ga4gh.cts.api.datasets.DatasetsTests;
+import org.ga4gh.methods.ExternalIdentifierQuery;
+import org.ga4gh.methods.SearchDatasetsRequest;
+import org.ga4gh.methods.SearchDatasetsResponse;
+import org.ga4gh.methods.SearchGenotypePhenotypeRequest;
+import org.ga4gh.methods.SearchGenotypePhenotypeResponse;
+import org.ga4gh.models.Dataset;
+import org.ga4gh.models.ExternalIdentifier;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.net.HttpURLConnection;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.StrictAssertions.fail;
+import static org.ga4gh.cts.api.Utils.catchGAWrapperException;
+
+/**
+ * Tests dealing with searching for datasets.
+ *
+ * @author Brian Walsh
+ */
+@Category(DatasetsTests.class)
+public class GenotypePhenotypeSearchIT {
+
+    private static Client client = new Client(URLMAPPING.getInstance());
+
+    /**
+     *  error if no parameters
+     * @throws AvroRemoteException if there's an unanticipated error
+     */
+    @Test
+    public void ensureCatchNoParametersSearch() throws AvroRemoteException {
+        try {
+            SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest.newBuilder().build();
+            SearchGenotypePhenotypeResponse response = client.genotypePhenotype.searchGenotypePhenotype(request);
+            fail("should have thrown error");
```

Comment:
Use `Utils.catchGAWrapperException` to catch the exception.  It will automatically fail if no exception is thrown by the thunk it wraps.